README.md

```diff
@@ -1,3 +1,66 @@
-# slashml-python-client
+# SlashML Python client
+[SlashML](https://www.slashml.com/)
+# Introduction
+## Overview 
+This is a Python client (SDK) for SlashML. It allows the user to use the apps available and active in [SlashML-Dashboard](https://www.slashml.com/dashboard). 
+The apps can be together in the same code. For example, if the user wants to transcribe an audio file and get a summary, they can call speechtotext followed by summarization.
 
-to be updated to guide users
+State-of-the-art AI models from several service providers are available. At SlashML, we also do the benchmarking on these models for the user. This will give them an idea of the best service provider for their application. For the full list of the available models thru SlashML, click [here](##availlable-service-providers)
```

Comment:
this will give who an idea for the best service provider. The user is reading the email. Write in third person, not second person.